charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml

```
@@ -30,6 +30,29 @@ spec:
         volumeMounts:
         - name: plugins
           mountPath: "/plugins"
+      - name: journald-config
+        image: {{ index .Values.global.images "alpine" }}
+        command:
+        - sh
+        - "-c"
+        - |
+          JOURNALD_PATH="/var/log/journal"
+          if [ ! -d ${JOURNALD_PATH} ]; then
+              JOURNALD_PATH="\/run\/log\/journal"
+          else
+              JOURNALD_PATH="\/var\/log\/journal"
+          fi
+
+          cp -rL /row-config/* /fluent-bit-config/
+          sed -i "s/%%JOURNALD_PATH%%/${JOURNALD_PATH}/g" /fluent-bit-config/input.conf
+          cat  /fluent-bit-config/input.conf
```

`sed` has some troubles when the substitution contains the delimiter as a symbol, I mean filepaths contain `/` and this breaks the sed command, therefore you can use different delimiter symbol, for example `|`. For more details see https://zhu45.org/posts/2016/Dec/21/environment-variable-substitution-using-sed/. 

And to avoid copying the entire directory, you can use stdout forwarding instead of in-place update.

```suggestion
          sed -e "s|%%JOURNALD_PATH%%|${JOURNALD_PATH}|g" /template-config/input.conf | tee /fluent-bit-config/input.conf
```